changelog.d/1-api-changes/WPB-4668-disable-defederation

```diff
@@ -0,0 +1 @@
+Remove de-federation (to avoid a scalability issue).
```

Comment:
Seems noisy for people reading the changelog to read about what hasn't changed.

Reply:
defederation has been introduced in 4.36.0.